Source/MDFastBinding/Private/BindingDestinations/MDFastBindingDestination_Property.cpp

```diff
@@ -96,14 +112,55 @@ void UMDFastBindingDestination_Property::SetupBindingItems()
 	Super::SetupBindingItems();
 
 	EnsureBindingItemExists(MDFastBindingDestination_Property_Private::PathRootName
-		, nullptr
+		, GetClass()->FindPropertyByName(GET_MEMBER_NAME_CHECKED(UMDFastBindingDestination_Property, ObjectProperty))
 		, LOCTEXT("PathRootToolTip", "The root object that has the property to set the value of. (Defaults to 'Self').")
 		, true);
 	EnsureBindingItemExists(MDFastBindingDestination_Property_Private::ValueSourceName
 		, PropertyPath.GetLeafProperty()
 		, LOCTEXT("ValueSourceToolTip", "The value to assign to the specified property"));
 }
 
+void UMDFastBindingDestination_Property::BindFieldNotify(UObject* SourceObject)
+{
+	UnbindFieldNotify();
+
+	const UE::FieldNotification::FFieldId FieldId = GetLeafFieldId(SourceObject);
+
+	if (FieldId.IsValid())
+	{
+		if (UObject* PropertyOwner = GetPropertyOwner(SourceObject))
+		{
+			if (INotifyFieldValueChanged* FieldNotify = Cast<INotifyFieldValueChanged>(PropertyOwner))
+			{
+				BoundFieldNotify = FieldNotify;
+				BoundFieldId = FieldId;
+			}
+		}
+	}
+}
+
+void UMDFastBindingDestination_Property::UnbindFieldNotify()
+{
+	BoundFieldNotify.Reset();
+	BoundFieldId = {};
+}
+
+UE::FieldNotification::FFieldId UMDFastBindingDestination_Property::GetLeafFieldId(UObject* SourceObject)
+{
+	if (const FFieldVariant Field = PropertyPath.GetLeafField())
+	{
+		if (UObject* PropertyOwner = GetPropertyOwner(SourceObject))
+		{
+			if (const INotifyFieldValueChanged* FieldNotify = Cast<INotifyFieldValueChanged>(PropertyOwner))
+			{
+				return FieldNotify->GetFieldNotificationDescriptor().GetField(PropertyOwner->GetClass(), Field.GetFName());
```

Comment:
Could this use `Field.GetOwnerClass()` instead of `PropertOwner->GetClass()`? (Since PropertyOwner isn't guaranteed to be a UObject)